src/routes/activity/service.js

```diff
@@ -0,0 +1,7 @@
+const repository = require('./repository')
+
+const createActivity = async activity => {
+    return await repository.insertActivity(activity)
```

Comment:
You can hardcode the auth object for now, since we don't have the OAuth middleware ready.
```suggestion
    // TODO: change the auth from context
    const auth = { userId: 1, deviceId: 1, deviceSerial: 'some_serial' }
    return await repository.insertActivity(activity, auth)
```